clang-tools-extra/clang-tidy/google/TodoCommentCheck.cpp

```diff
@@ -45,7 +54,7 @@ class TodoCommentCheck::TodoCommentHandler : public CommentHandler {
 private:
   TodoCommentCheck &Check;
   std::string User;
-  llvm::Regex TodoMatch;
+  std::list<std::string> TodoMatches;
```

Comment:
This is horribly inefficient, if you need multiple regexes use a vector of `llvm::regex`.
However for this patch there doesn't need to be 2 regexes as you can match both cases with just the one

Reply:
Hi @njames93, thanks for the review. I've updated the PR to initialize the regexes in the constructor.

I had thought about combining the regex, but I had concluded that it would make it very difficult to read / update / maintain.

Please take a look at the current version, thanks!